src/db.ts

```diff
@@ -8,12 +8,14 @@ import { TLRUCache } from '@thi.ng/cache';
 import { SingleValueData } from 'lightweight-charts';
 
 import { getObj, setObj } from './storage';
+import { TWAP_RETRIES } from './api/trade';
 
 export const TRADE_DATA_OPTS = { ttl: 1000 * 60 * 60 };
 
 export const DEFAULT_TRADE_CONFIG: TradeConfig = {
   slippage: '1',
   slippageTwap: '3',
+  maxRetries: TWAP_RETRIES,
```

Comment:
Use "5" here and get rid of api/trade import .. There is static function in api/trade getTwapTxFee which using TWAP_RETRIES. U have to use custom retries value over there as well in order to calculate correct fees for twap ;)